src/ot_croissant/assets/recordset.json

```diff
@@ -140,54 +144,40 @@
     "description": "Name of the biosample"
   },
   {
-    "id": "biosample/description",
-    "description": "Description of the biosample"
+    "id": "biosample/children",
+    "description": "Direct child biosample IDs in the ontology"
   },
   {
-    "id": "biosample/xrefs",
-    "description": "Cross-reference IDs from other ontologies"
+    "id": "biosample/descendants",
+    "description": "List of descendant biosample IDs in the ontology"
   },
   {
-    "id": "biosample/synonyms",
-    "description": "List of synonymous names for the term"
+    "id": "biosample/description",
+    "description": "Description of the biosample"
   },
   {
     "id": "biosample/parents",
     "description": "Direct parent biosample IDs in the ontology"
   },
   {
-    "id": "biosample/ancestors",
-    "description": "List of ancestor biosample IDs in the ontology"
-  },
-  {
-    "id": "biosample/children",
-    "description": "Direct child biosample IDs in the ontology"
-  },
-  {
-    "id": "biosample/descendants",
-    "description": "List of descendant biosample IDs in the ontology"
+    "id": "biosample/synonyms",
+    "description": "List of synonymous names for the term"
   },
   {
-    "id": "colocalisation_coloc/leftStudyLocusId",
-    "description": "Study-locus identifier for left-side overlapping signal",
-    "foreign_key": "credible_set/studyLocusId"
+    "id": "biosample/xrefs",
+    "description": "Cross-reference IDs from other ontologies"
   },
   {
-    "id": "colocalisation_coloc/rightStudyLocusId",
-    "description": "Study-locus identifier for right-side overlapping signal",
-    "foreign_key": "credible_set/studyLocusId"
+    "id": "colocalisation_coloc/betaRatioSignAverage",
```

Comment:
Colocalisation Table ids outdated?

Reply:
You're right, I have updated the distribution, but not the recordsets.